pkg/cli/admin/release/info.go

```diff
@@ -783,6 +818,21 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 				errs = append(errs, err)
 				return true, nil
 			}
+			// If using an alternative image source, have to update all tags
+			if replacedRef || alternativeSource {
+				for _, tag := range is.Spec.Tags {
+					// If can't get digest ID, skip this tag, this happens when user has built a payload by
+					// replacing component images in the release with a new image (unsupported configuration)
+					tagRef, err := imagereference.Parse(tag.From.Name)
+					// if err != nil, skip this tag
```

Comment:
I think I missed it yesterday, fail early and try to avoid unnecessary blocks:
```golang
if err != nil {
    continue
}
ref.Ref.ID = tagRef.ID
...
```

Reply:
fixed